deploy/osd-machine-configuration-ami/config.yaml

```diff
@@ -0,0 +1,6 @@
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+  matchExpressions:
+    - key: hive.openshift.io/version-major-minor
+      operator: In
+      values: ["4.17", "4.18"]
```

Comment:
Per slack discussion, if we move to a "notIn" [4.0 to 4.16] rule, we don't need to revisit this adding every new Y version in the future.